### PR TITLE
Update homepage banner.

### PR DESCRIPTION
**Pull request by mikelehen**
image

### CODE REVIEW

File: src/components/Banner/SpringSurgeBanner.tsx [hunk 1]
```diff
@@ -13,22 +14,28 @@ const trackClick = () => {
...
+  const millionVaccinations = Math.round(totalVaccinationsInitiated / 1000000);
+
+  const ManyStatesLink = (
     <HashLink
       to="#explore-hospitalizations"
       onClick={trackClick}
       scroll={(element: HTMLElement) => scrollWithOffset(element, -80)}
     >
-      several states
+      many states
```
Comment (mikelehen) [inline src/components/Banner/SpringSurgeBanner.tsx:27]: I can revert if you're not a fan, but about half of the states that have rising cases also have rising hospitalizations so "many" feels appropriate.

File: src/screens/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -67,15 +67,15 @@ export default function HomePage() {
...
     'MN',
     'NJ',
-    'WV',
-    'CT',
-    'IL',
-    'MA',
-  ]; // Updated on 29 March 2021
+    'PA',
+    'PR',
+  ]; // Updated on 7 April 2021 -- this is an illustrative list but not exhaustive
```
Comment (mikelehen) [inline src/screens/HomePage/HomePage.tsx:78]: We could add several more states, but the graph just gets noisy, hurts performance (we're downloading the API endpoint for each state), and also some of the "low but rising" states just don't really pop when on a normalized graph with the other ones, so I don't think we should try to make this be an exhaustive list...

**Comment by mikelehen** [comment]
[Update Banner to "Daily new cases are rising in half of the states, ..."](https://trello.com/c/KnVsTuI6/1238-update-banner-to-daily-new-cases-are-rising-in-half-of-the-states)

File: src/components/Banner/SpringSurgeBanner.tsx (hunk 1 above)
Comment (BrettBoval) [reply src/components/Banner/SpringSurgeBanner.tsx:27]: I like it. "Many" does feel appropriate.

File: src/screens/HomePage/HomePage.tsx (hunk 2 above)
Comment (BrettBoval) [reply src/screens/HomePage/HomePage.tsx:78]: Agreed that we can treat this as a representative, not exhaustive, list.

**Review by BrettBoval — APPROVED** [review]

**Pull request merged**